DESCRIPTION

```diff
@@ -20,13 +20,16 @@ Imports:
     miniUI (>= 0.1.1),
     shiny (>= 0.13.0),
     shinyjs (>= 0.5.2),
-    utils
+    utils,
+    grDevices
 Suggests:
     knitr (>= 1.7),
     rmarkdown,
-    rstudioapi (>= 0.5)
+    rstudioapi (>= 0.5),
+    testthat,
```

Comment:
I would keep all packages that are used in testing in `Suggests`. It's fine if they aren't reverse dependency checked. I strongly believe that a package should have as few dependencies as possible, and anything required for running the tests is not essential. With regards to putting tests in `Suggests`, looks like [Hadley agrees](http://r-pkgs.had.co.nz/description.html#dependencies)